electron/src/main.ts

```diff
@@ -352,7 +364,7 @@ class ElectronWrapperInit {
     };
 
     app.on('web-contents-created', (webviewEvent, contents) => {
-      switch (contents.getType()) {
+      switch ((contents as any).getType()) {
```

Comment:
This is Electron code. Should be typed properly above.

Reply:
Missing type again.